src/markers.json

```diff
@@ -0,0 +1,14 @@
+{
+  "af0bbb101d58b4a49a37351d0d20705a": "markers/5d5147d6960019.03296979-1.png",
+  "a942c603f482cae9f47b557c9d860552": "markers/5d362e60939375.03477136-1.png",
+  "4753397cd46f823b20deabb7de8eaeb5": "markers/5cead0f685f794.81367704-1.png",
+  "4377494c7736539173829020be70ea5b": "markers/5ceaec838bbe95.07755000-1.png",
+  "f1ffee2f61b9c96a2565c6adc5b13d5a": "markers/5ceffb08ce69c0.03408143-1.png",
+  "8b92d8150d81e61165814c839473cfa5": "markers/5cf0044ad0ce30.91228640-1.png",
+  "420bbbb08a8ebb46c5f7134ccde94cd8": "markers/5d362ff4a66304.92462847-1.png",
+  "476e010383589f667b226a50d04b1a95": "markers/5d3633efc5f844.79469799-1.png",
+  "22501420b5d426fc66ba76bb4b15d70e": "markers/5d514785cc23e0.19218416-1.png",
+  "2941e4d4668208625f0e3e75d202e001": "markers/5e0250cb76c988.78082998-1.png",
+  "7e0079bb46816067e6d9a99f9bee21a8": "markers/5e6e2df31d0b55.94376998-1.png",
+  "391c2e53db5f84179f0466597b186287": "markers/5e1502f687e0f3.24498958-1.png"
```

Comment:
These keys are referenced in the "icon" prop in `points.json`.

We might want to rename these file hashes to human readable strings. Or not.